From b27229be5aa71e074bd08a6a0b086001844a122f Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 19:43:08 +0800
Subject: [PATCH 0118/5242] MLK-9637-1 usb: test: define otg_srp_reqd wIndex
 byte value

commit  23ec70f9595e12764acb367e00eee095da9b8ce6 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds otg_srp_reqd wIndex high byte value for otg test mode.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 34660f7aa14972630622406b9dbfc7a106d8d15d)
(cherry picked from commit c5128f9bafaa37a8e63989032372a5ff36dc82c4)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/usb/ch9.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/usb/ch9.h b/include/uapi/linux/usb/ch9.h
index d5a5cae..e5414b6 100644
--- a/include/uapi/linux/usb/ch9.h
+++ b/include/uapi/linux/usb/ch9.h
@@ -143,6 +143,7 @@
 #define	TEST_SE0_NAK	3
 #define	TEST_PACKET	4
 #define	TEST_FORCE_EN	5
+#define	TEST_OTG_SRP_REQD	6
 
 /* Status Type */
 #define USB_STATUS_TYPE_STANDARD	0
-- 
1.7.9.5

